basic/settings/index.rst

```diff
@@ -2173,6 +2173,26 @@ TINYMCE_DEFAULT_CONFIG
 U
 =
 
+UI_REQUIRED_FIELDS
+------------------
+If this option is enabled, the input selected (we are referring about the onece present in the Optional Metadata tab  on Metadata Wizard) will become Mandatory.
```

Comment:
Maybe better: 

If this option is enabled, the input selected (we are referring to the one present in the optional Metadata-Tab on the Metadata-Wizard) will become mandatory.

(small typo)

Reply:
Hey @t-book thanks for the comment. PR updated :)